pkg/agent/handler/managedapplication_test.go

```diff
@@ -134,7 +134,7 @@ func TestManagedApplicationHandler(t *testing.T) {
 			if tc.teamName != "" {
 				cm.AddTeam(team)
 			}
-			handler := NewManagedApplicationHandler(p, cm, c)
+			handler := NewManagedApplicationHandler(p, cm, c, config.DiscoveryAgent)
```

Comment:
Can you please add a test for the trace agent resource handling

Reply:
Done